indico/modules/events/sessions/templates/management/_sessions_list.html

```diff
@@ -0,0 +1,111 @@
+                                       data-method="PATCH" data-href="{{ url_for('.change_colors', session) }}">
+                                    </a>
+                                </span>
+                            </div>
+                        </td>
+                        <td class="i-table">
+                            <div class="group right">
+                                <a class="icon-edit highlight edit-session"
+                                   title="{% trans %}Edit this session{% endtrans %}"
+                                   data-title="{% trans title=session.title %}Edit session {{ title }}{% endtrans %}"
+                                   data-href="{{ url_for('.modify_session', session) }}"></a>
+                                <a class="icon-users highlight session-persons-list"
+                                   data-title="{% trans title=session.title %}List of session persons{% endtrans %}"
+                                   data-href="#"></a>
+                                <a class="icon-time highlight"
+                                   title="{% trans %}Go to this session timetable{% endtrans %}"
+                                   data-href="{{ url_for('event_mgmt.sessionModifSchedule', event, sessionId=session.legacy_mapping.id) }}"
+                                   data-method="POST"></a>
+                                <a class="icon-remove highlight js-remove-session"
+                                   title="{% trans %}Remove this session{% endtrans %}"
+                                   data-method="DELETE"
+                                   data-href="{{ url_for('.delete_session', session)}}"
+                                   data-confirm="{% trans title=session.title %}Do you really want to delete session '{{ title }}'?{% endtrans %}"></a>
+                            </div>
+                        </td>
+                    </tr>
+                    {% for block in session.blocks -%}
+                        <tr class="i-table session-blocks-row" style="display: none;">
+                            <td class="i-table" colspan="2"></td>
+                            <td class="i-table">
+                                {{- block.title or 'Default empty block' -}}
+                            </td>
+                            <td class="i-table">{{ block.duration }}</td>
+                            <td class="i-table">{{ block.background_color }}</td>
+                            <td class="i-table"></td>
+                            <td class="i-table"></td>
+                        </tr>
+                    {%- endfor %}
+                {%- endfor %}
+            </tbody>
+        </table>
+    {%- else %}
+        {%- call message_box('info') -%}
+            {% trans %}There are no sessions yet.{% endtrans %}
+        {%- endcall %}
+    {%- endif %}
+    <script>
+        $('.js-color-switch').colorswitch({
+            defaultColors: {{ default_colors | tojson }},
```

Comment:
I remember I had problems when rerendering the list but since I changed event handlers I will give   it another try.